transport/transport.go

```diff
@@ -213,6 +213,9 @@ type Stream struct {
 	// the status received from the server.
 	statusCode codes.Code
 	statusDesc string
+	// rstStream is a flag that is true when a RST stream frame
+	// is sent to the server signifying that this stream is closing.
```

Comment:
rstStream indicates whether a RSTStream **needs to be sent** to the server to signify that this stream is closing.

"is sent" is a bit confusing.